test/end-to-end/router_test.go

```diff
@@ -1324,6 +1324,10 @@ func createAndStartRouterContainerExtended(dockerCli *dockerClient.Client, maste
 		fmt.Sprintf("ROUTER_BIND_PORTS_AFTER_SYNC=%s", strconv.FormatBool(bindPortsAfterSync)),
 		fmt.Sprintf("ROUTER_ENABLE_INGRESS=%s", strconv.FormatBool(enableIngress)),
 		fmt.Sprintf("NAMESPACE_LABELS=%s", namespaceLabels),
+		fmt.Sprintf("ROUTER_CONFIG_MANAGER=haproxy-manager"),
+		fmt.Sprintf("ROUTER_DYNAMIC_SERVER_PREFIX=_test-dynamic"),
+		fmt.Sprintf("ROUTER_MAX_DYNAMIC_SERVERS=3"),
```

Comment:
These should have reasonable defaults.  I'd prefer if your testing required only ROUTER_CONFIG_MANAGER unless there's a good reason not to.

Reply:
ok - i'll set it to only the defaults.